.github/workflows/build.yaml

```diff
@@ -0,0 +1,70 @@
+name: build
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.13.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build x86_64 container into library
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          outputs: "type=docker,push=false"
+          platforms: linux/amd64
+          tags: |
+            ghcr.io/openfaas/cron-connector:${{ github.sha }}
+
+      - name: Build multi-arch containers for validation only
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          outputs: "type=image,push=false"
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          tags: |
+            ghcr.io/openfaas/cron-connector:${{ github.sha }}
+      # Todo - load the image into Kind before running tests
+      # otherwise, this just tests the chart with images that 
+      # have already been built and pushed in an earlier job
+      - name: get tools
```

Comment:
We don't need all this, can you delete it?

Reply:
done.